README.md

```diff
@@ -71,6 +72,7 @@ For openssl 1.1
     cd uadk_engine
     autoreconf -i
     ./configure --libdir=/usr/local/lib/engines-1.1/ [--enable-kae]
+    sed -i "s/hardcode_into_libs=yes/hardcode_into_libs=no/g" ./libtool
```

Comment:
any other project has this requirement? 
how about qat.
user manually set " sed -i  " seems tricky.

Reply:
This can be done by 
--- a/configure.ac
+++ b/configure.ac
@@ -6,6 +6,7 @@ AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS([config.h])

 AC_PROG_CC
+LT_INIT([disable-shared])
 LT_INIT

both uadk and uadk_engine, though uadk default value is no